src/main/java/com/faforever/client/preferences/ui/SettingsController.java

```diff
@@ -219,6 +232,24 @@ public void initialize() {
     passwordChangeErrorLabel.setVisible(false);
   }
 
+  private void configureTimeSetting(Preferences preferences) {
+    timeComboBox.setItems(FXCollections.observableArrayList(TimeInfo.getAllDisplayNames()));
+    timeComboBox.setOnAction(event -> {
+      newTimeFormatSelected(event);
+    });
+    timeComboBox.setDisable(false);
+    timeComboBox.setFocusTraversable(true);
+    timeComboBox.getSelectionModel().select(TimeInfo.getIndexByDisplayName(preferences.getChat().getUkTime()));
+  }
+
+  private void newTimeFormatSelected(Event event) {
+    logger.info("newTimeFormat is " + timeComboBox.getValue().toString());
+    Preferences preferences = preferencesService.getPreferences();
+    preferences.getChat().setUkTime(timeComboBox.getValue().toString());
+    preferencesService.storeInBackground();
+    logger.info("saving.....Time Format");
```

Comment:
`logger.info("Saving new time format");`

But actually one log is enough, because the preferences service also logs already.